torchtune/modules/transformer.py

```diff
@@ -478,7 +489,14 @@ def reset_caches(self):
         for layer in self.layers:
             layer.reset_cache()
 
-    @torch.compiler.disable
+    def get_output_weight(self) -> torch.Tensor:
+        """Returns the output weight matrix."""
+        return self.output.weight
+
+    def set_skip_output_layer(self, skip: bool) -> None:
```

Comment:
I would opt for a slightly longer description here of when and why you would want to use this.